### ISSUE TITLE
How to train with multi-gpu

### ISSUE THREAD

**Issue by sicilyliulc** [iperov/DeepFaceLab#278]
i have 8 gpu，i want to train with multi-gpu，what does models_list mean，i modify code ，but can not do multi-gpu。
    def to_multi_gpu_model_if_possible (self, models_list):
       if len(self.device_config.gpu_idxs) > 1:
           #make batch_size to divide on GPU count without remainder
           self.batch_size = int( self.batch_size / len(self.device_config.gpu_idxs) )
           if self.batch_size == 0:
               self.batch_size = 1
           self.batch_size *= len(self.device_config.gpu_idxs)

           result = []
           for model in models_list:
               for i in range( len(model.output_names) ):
                   model.output_names = 'output_%d' % (i)
               result += [ nnlib.keras.utils.multi_gpu_model( model, self.device_config.gpu_idxs ) ]

           return result
       else:
           return models_list

**Comment by iperov** [comment]
no easy way to train by multi gpu

keras.utils.multi_gpu_model cannot be used in training mode.
